R/nest.R

```diff
@@ -106,39 +151,52 @@ nest <- function(.data, ..., .names_sep = NULL, .key = deprecated()) {
       i = "Did you want `{(.key)} = {cols_fixed_label}`?"
```

Comment:
Should we switch this to `lifecycle_warn(always = TRUE)`?  Or maybe even make it defunct? It was deprecated in tidyr 1.0.0 (August 2019), so it's been >3 years with a warning on every use.

Reply:
Hmm I'll fiddle with that in a follow up PR